### PR TITLE
Add missing JAVA_SPEC_VERSION to CXXFLAGS

### PR DESCRIPTION
**Pull request by jdekonin**
Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

**Comment by jdekonin** [comment]
fyi @pdbain-ibm @pshipton

File: runtime/makelib/targets.mk.ftl [hunk 1]
```diff
@@ -95,6 +95,7 @@ ifeq ($(UMA_TARGET_TYPE),EXE)
 endif
 
 CFLAGS+=-DJAVA_SPEC_VERSION=$(VERSION_MAJOR)
+CXXFLAGS+=-DJAVA_SPEC_VERSION=$(VERSION_MAJOR)
```
Comment (keithc-ca) [inline runtime/makelib/targets.mk.ftl]: Perhaps it should be in `CPPFLAGS` too?

Comment (jdekonin) [reply runtime/makelib/targets.mk.ftl]: I should've seen that.  Done.

**Comment by keithc-ca** [comment]
jenkins test sanity

**Pull request merged**